lib/pod/command/trunk/push.rb

```diff
@@ -36,6 +36,8 @@ def self.options
              'This takes precedence over a .swift-version file.'],
             ['--skip-import-validation', 'Lint skips validating that the pod can be imported'],
             ['--skip-tests', 'Lint skips building and running tests during validation'],
+            ['--synchronous', \
+             'If validation depends on other pods pushed in the last ten minutes, synchronize'],
```

Comment:
I'd prefer to avoid explicit mentions of time duration in this as it implies some implementation details about the CDN / trunk infra